packages/core/mesh/teleport/src/muxing/balancer.ts

```diff
@@ -69,36 +71,9 @@ export class Balancer {
   }
 
   pushData(data: Uint8Array, trigger: Trigger, channelId: number) {
-    const noCalls = this._sendBuffers.size === 0;
-
-    if (!this._channels.includes(channelId)) {
-      throw new Error(`Unknown channel ${channelId}`);
-    }
-
-    if (!this._sendBuffers.has(channelId)) {
-      this._sendBuffers.set(channelId, []);
-    }
-
-    const sendBuffer = this._sendBuffers.get(channelId)!;
-
-    const chunks = [];
-    for (let idx = 0; idx < data.length; idx += MAX_CHUNK_SIZE) {
-      chunks.push(data.subarray(idx, idx + MAX_CHUNK_SIZE));
-    }
-
-    chunks.forEach((chunk, index) => {
-      const msg = encodeChunk({
-        chunk,
-        channelId,
-        dataLength: index === 0 ? data.length : undefined,
-      });
-      sendBuffer.push({ msg, trigger: index === chunks.length - 1 ? trigger : undefined });
-    });
-
+    this._enqueueChunk(data, trigger, channelId);
     // Start processing calls if this is the first call.
```

Comment:
remove comment?

Reply:
whoops, that one eluded my `git add -p`. more cleanup in general to do if this works.